src/components/VideoDetail/VideoDetail.component.jsx

```diff
@@ -1,92 +1,70 @@
 import React, { useState, useEffect } from 'react';
-import { useParams } from 'react-router-dom';
 import { VideoFrame, UnorderedList, LinkVideo } from './styled';
 import Grid from '@material-ui/core/Grid';
 
-import youtube from '../../api/youtube';
+import { youtube } from '../../api/youtube';
 
-const handleFetch = (criteria, setRelated) => {
-  youtube
-    .get('/search', {
-      params: {
-        part: 'snippet',
-        relatedToVideoId: criteria,
-        type: 'video',
-      },
-    })
-    .then(
-      (response) => {
-        setRelated(response);
-      },
-      (error) => {
-        console.log('error: ', error);
-      }
-    );
-};
-
-const VideoDetail = (props) => {
-  const params = useParams();
-  const videoId = params.videoId;
-  const videoName = params.name;
+const VideoDetail = ({ videoId, videoName }) => {
   const [related, setRelated] = useState([]);
   useEffect(() => {
-    youtube
-      .get('/search', {
-        params: {
-          part: 'snippet',
-          relatedToVideoId: videoId,
-          type: 'video',
-        },
+    const response = youtube
+      .related(videoId)
+      .then((data) => {
+        setRelated(data);
       })
-      .then((response) => {
-        console.log(JSON.stringify(response));
-        setRelated(response);
+      .catch((err) => {
+        console.log(err);
```

Comment:
if it's an error message, even for logs could be of error type, try using it also:
```suggestion
        console.error(err);
```